modules/nickserv.cpp

```diff
@@ -119,9 +141,18 @@ class CNickServ : public CModule {
             MCString msValues;
             msValues["password"] = GetNV("Password");
             PutIRC(CString::NamedFormat(GetNV("IdentifyCmd"), msValues));
+        } else if(Nick.NickEquals(sNickServName) && sMessage.find("Password accepted") != CString::npos && m_bJoinAfterIdentified) {
```

Comment:
Ok, i took a look around and saw that every irc network I visited which uses nickserv has "Password accepted" in their message and it's also default in unrealircd
But you're right, there are for sure networks that don't do that so I'm prepared if someone has problems with this option.